commands/permission_role_test.go

```diff
@@ -465,3 +465,50 @@ Permissions   edit_brand
 		s.Require().Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestListRoleCmd() {
+	s.Run("List roles", func() {
+		printer.Clean()
+		printer.SetFormat(printer.FormatPlain)
+		defer printer.SetFormat(printer.FormatJSON)
```

Comment:
Is default format is `JSON`? I don't think we need to set it back to json.

Reply:
followed similar structure to `TestShowRoleCmd`